File: DatabaseDemo/app/src/main/java/com/dilip/databasedemo/MainActivity.java
package com.dilip.databasedemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        MyDBHelper dbHelper = new MyDBHelper(this);

        dbHelper.addContact("Dilip", "5556454656");
        dbHelper.addContact("Raman", "4545451545");
        dbHelper.addContact("Darshan", "31433444");
        dbHelper.addContact("Vraj", "6848768482");
        dbHelper.addContact("Dhruv", "6449599455");



    }
}

File: app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import android.os.Bundle
import android.view.View.INVISIBLE
import android.view.View.VISIBLE
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        val greetingTextView = findViewById<TextView>(R.id.HelloWorld)
        val inputField = findViewById<EditText>(R.id.tvText)
        val submitButton = findViewById<Button>(R.id.button)
        val offersButton = findViewById<Button>(R.id.btnOffers)

        var enteredName = ""
        submitButton.setOnClickListener{
            enteredName = inputField.text.toString()
            if (enteredName == "") {
                offersButton.visibility = INVISIBLE
                greetingTextView.text = ""
                Toast.makeText(this@MainActivity, "Please, Enter your name", Toast.LENGTH_SHORT).show()
            } else {
                val message = "Welcome $enteredName"
                greetingTextView.text = message
                inputField.text.clear()
                offersButton.visibility = VISIBLE
            }

            offersButton.setOnClickListener {
                val intent = Intent(this, SecondActivity::class.java)
                intent.putExtra("USER", enteredName)
                startActivity(intent)
            }




        }
    }
}